### DISCUSSION TITLE
Hats off to you sir

### DISCUSSION THREAD

**Discussion by r0b3rtf1sh3r** [cem-akkaya/LightAwareness#4] [General]
thank you for creating this, it fit my needs perfectly for detecting light levels driving "solar panel" regeneration style effect in my game
easily just added to plugins folder. configured variables. and debug readings to adjust factors. and BAM
works perfectly. 

one suggestion. perhaps include a way to determine which actor is checked for light. that way you could check the collision capsule or any other thing instead of character/actor =)

**Comment by cem-akkaya** [comment]
My pleasure. 
Sure makes sense, it can basically return the owner actor for ease of use which is super simple.

however at a second thought in large systems like yours that the component used many many times, there could be a mini subsystem to extend its functionality. Get all registered light awareness components etc. 

So I will mark this for @todo and thanks for feedback

**Reply by cem-akkaya** [reply]
Opened a new issue here to do. https://github.com/cem-akkaya/LightAwareness/issues
